tf-deployer/spec/calico-v3.0.yaml

```diff
@@ -0,0 +1,443 @@
+# Calico Version v3.0.7
```

Comment:
In the startup script, you install calico from URL. Why are you checking in this file?

Reply:
This version is modified, I need it to existed somewhere. If you think contain it in this repo is not a good choice, I can create a new repo.

Reply:
Then my question is why install from the URL in the startup script if we need to use a modified version?

What are the modifications? We should document them in the README or somewhere more appropriate as part of this change.

Reply:
We don't have to install from the URL. So do you prefer install from URL or from local?

Reply:
I guess I'm confused because you are checking in a file but not using it. We would like to have the repo have the current state of world that the user can run.

Does installing from this file break the cluster? Does installing from the URL?

If you are still developing this file or installing from it breaks the cluster, please do not check it in.